packages/odyssey/CHANGELOG.md

```diff
@@ -5,7 +5,43 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.6.0] - 2020-10-01
+## [0.7.0] - 2020-11-18
+
+### Added
+- `vuepress-theme-odyssey` is a new private package containing our brand new docs theme
+- adds `prism-theme-odyssey` package for styling code blocks
```

Comment:
These are [docs] only?

Reply:
@arnoldsandoval-okta and I are currently discussing collapsing all docs packages into a single `odyssey-website` package. For now, these are distinct on purpose as we've had requests for separate Docs content, VuePress Theme, and Prism theme packages. 

Prism (or its replacement) will likely stay distinct afterward as it can be applied to non-docs code as well.

Reply:
Can we move these `-theme-` parts to the bottom as well?